packages/fxa-content-server/tests/functional/settings_v2/lib/helpers.js

```diff
@@ -2,19 +2,24 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const intern = require('intern').default;
```

Comment:
This isn't necessary.  Why was this added?  I just want to make sure that I didn't miss or misunderstand something.